src/main/java/uk/gov/hmcts/et/common/model/ccd/Et1CaseData.java

```diff
@@ -49,5 +49,8 @@ public class Et1CaseData {
     private List<DocumentTypeItem> servingDocumentCollection;
     @JsonProperty("otherTypeDocumentName")
     private String otherTypeDocumentName;
-
+    @JsonProperty("claimantAddress")
+    private String claimantAddress;
```

Comment:
could you maybe use ClaimantType -> ClaimantAddress instead rather than creating a new property?
same for respondent - respondentCollection -> respondent -> respondentAddress?

is there a reason you are creating new properties?